.github/workflows/test.yml

```diff
@@ -364,20 +364,21 @@ jobs:
         # ipfamily: ["ipv4", "ipv6", "dualstack"]
         # disable-snat-multiple-gws: ["noSnatGW", "snatGW"]
         # second-bridge: ["2br", "1br"]
+        # cluster-manager-mode: ["inmaster", "service"]
```

Comment:
So what i referred to as `zone-mode`seems to be that interconnect flag, and is only set to `true` when `cluster-manager-mode` is set to `service` just the same as I expressed above.

There is many modes in ovnkube master and we probably don't want to test all possible combinations, just the ones that relate to a configuration we want to support. That's what I think addressing each mode directly in a specific configuration parameter doesn't really scale in the long run and that it would rather be easier to have a parameter that addresses a global concept related to a congiguration topology we want to support.

Also is difficult to relate to the service term in this context.

One other final suggestion: `network-manager-mode` value `local` vs `cluster`.

Anyway, if there is no straight thing that looks better for everyone, let's keep what you have,